plugins/woocommerce/src/Admin/API/OnboardingFreeExtensions.php

```diff
@@ -68,12 +73,29 @@ public function get_items_permissions_check( $request ) {
 	/**
 	 * Return available payment methods.
 	 *
-	 * @param \WP_REST_Request $request Request data.
+	 * @param WP_REST_Request $request Request data.
 	 *
-	 * @return \WP_Error|\WP_REST_Response
+	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_available_extensions( $request ) {
-		return RemoteFreeExtensions::get_extensions();
+		$extensions = RemoteFreeExtensions::get_extensions();
+		/**
+		* Remove Jetpack when woocommerce_suggest_jetpack is false.
```

Comment:
Just a note: we use these comments to auto-generate docs, as well as lists of new filters introduced in a release. It'd be helpful in the future if the description more completely described the filter, such as "When false, Jetpack is removed from the list of suggested extensions."

No changes immediately required here, but please keep this in mind when adding these comments to future filters and when reviewing PRs that include new filters.

Reply:
/cc @chihsuan  and @adrianduffell as reviewers as well

Reply:
Thank you! @jonathansadowski

Reply:
cc @woocommerce/ghidorah for the rest of the team

Reply:
Great callout, @jonathansadowski!